ReactWindows/ReactNative.Shared/UIManager/LayoutAnimation/StoryboardObservable.cs

```diff
@@ -26,8 +30,13 @@ public IDisposable Subscribe(IObserver<Unit> observer)
             _storyboard.Begin();
 
             return Observable.FromEventPattern<object>(
+#if WINDOWS_UWP
                 h => _storyboard.Completed += h,
                 h => _storyboard.Completed -= h)
+#else
+                h => _storyboard.Completed += new EventHandler(h),
+                h => _storyboard.Completed -= new EventHandler(h))
+#endif
```

Comment:
Right, that's why you use the Observable.FromEventPattern (non-generic) overload, which defaults to using EventHandler (non-generic).  I'd rather do that than cast the EventHandler -> EventHandler<object>